CHANGELOG.md

```diff
@@ -5,12 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [2.9.5] - 2024-05-06
+## [2.9.5] - 2024-05-17
```

Comment:
Please do not modify previously released versions.
To create a new block just write:

```suggestion
## [unreleased]
```

We'll adapt the number and date when we deploy the release.